config/forms/2021_2022.json

```diff
@@ -1070,30 +1070,42 @@
                 }
               }
             },
-            "fixed_term_tenancy": {
+            "tenancy_type": {
               "header": "",
               "description": "",
               "questions": {
-                "tenancylength": {
-                  "check_answer_label": "If the main tenancy is a fixed term tenancy, please provide the length of the fixed term (to the nearest year) excluding any starter/introductory period",
-                  "header": "If fixed-term, what is the length of the fixed-term tenancy after any starter period?",
-                  "hint_text": "To the nearest year",
-                  "type": "numeric",
-                  "min": 0,
-                  "max": 150,
-                  "step": 1,
-                  "width": 4
+                "tenancy": {
+                  "check_answer_label": "Type of main tenancy",
+                  "header": "What is the type of tenancy?",
+                  "hint_text": "",
+                  "type": "radio",
+                  "answer_options": {
+                    "0": "Secure (including flexible)",
+                    "1": "Assured",
+                    "2": "Assured Shorthold",
+                    "3": "Licence agreement (almshouses only)",
+                    "4": "Other"
```

Comment:
Should be alphabetically ordered (with ‘Other’ last)

Reply:
Got it, will fix this one 😄

Reply:
This should be fixed now 😄